.github/workflows/aselo_beta.yml

```diff
@@ -28,8 +69,8 @@ jobs:
       - name: Fill secret.ts
         run: |
           cat <<EOT >> ./private/secret.ts
-          export const API_KEY = '${{ secrets.IP_FIND_API_KEY }}';
-          export const SERVERLESS_URL = '${{ secrets.BETA_SERVERLESS_URL }}';
+          export const API_KEY = '$/beta/webchat/${{ inputs.account-sid }}/api-key';
+          export const SERVERLESS_URL = '$/beta/webchat/${{ inputs.account-sid }}/serverless-url';
```

Comment:
I *think* the `env_variable_name` argument for `marvinpinto/action-inject-ssm-secrets@latest` tells it to set the value of the secret stored in SSM to that variable name so these are going to look like:

```yaml
          export const API_KEY = '${{ env.IP_FIND_API_KEY }}';
          export const SERVERLESS_URL = ${{ env.SERVERLESS_URL }};
```